tests/snmp/test_snmp_interfaces.py

```diff
@@ -142,7 +157,6 @@ def test_snmp_interfaces(localhost, creds_all_duts, duthosts, enum_rand_one_per_
     for po_name in config_facts.get('PORTCHANNEL', {}):
         assert po_name in snmp_ifnames, "PortChannel not found in SNMP facts."
 
-@pytest.mark.bsl
```

Comment:
>pytest.mark.bsl [](http://example.com/codeflow?start=1&length=15)

Why remove this?

Reply:
Need not be removed, I have added it back.